base/applications/rapps/loaddlg.cpp

```diff
@@ -640,7 +651,10 @@ unsigned int WINAPI CDownloadManager::ThreadFunc(LPVOID param)
         }
 
         if (!hOpen)
+        {
+            MessageBox_LoadString(hMainWnd, IDS_UNABLE_TO_INTACCESS);
```

Comment:
You can use some API function that translates error code into localized text message, so you don't need to add all these messages in app.

Reply:
Which API do you have in mind ? Most messages here are implementation-specific, not simply GetLastError(). Did I miss something ?

Reply:
FormatMessage with HMODULE.

Reply:
Sorry but i'm missing the purpose.... what would be the benefit here ? 1 single IDS_UNABLE_TO like "Error occured : %d" ? Tailored error message looks more explicit for end user as cause may differ (configuration of path, disk capacity, proxy settings....)

Reply:
> 1 single IDS_UNABLE_TO like "Error occured : %d" ?

It's applicable to all error strings you added.

Reply:
Sorry but I still don't undersand what you expect... In any case, i'll have the same number of strings to be translated and more code.... May I have a very simple PoC of what you suggest ?

Reply:
Same design solution as for https://github.com/reactos/reactos/pull/2653